charts/resoto/values.yaml

```diff
@@ -50,6 +51,12 @@ resotoworker:
 
 
 resotometrics:
+  image:
+    # -- Image repository
+    repository: somecr.io/someengineering/resotometrics
+    # -- Overrides the image tag whose default is the chart appVersion.
+    tag: ""
+  imagePullPolicy: IfNotPresent
```

Comment:
This does not have any effect or? (defined on line 11)

Reply:
Good catch @aquamatthias I missed it